scripts/fetchdep.sh

```diff
@@ -19,4 +22,4 @@ then
 fi
 
 echo "Checking out develop branch"
```

Comment:
I think this should also change to something like:

```suggestion
echo "Checking out default branch $defbranch"
```